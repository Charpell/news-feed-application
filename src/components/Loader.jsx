import React from 'react';

/**
 * Spinner when page is waiting to laod
 * @returns {void}
 */
function Loader() {
  return (
    <div className="load">
      Loading <i
      className="fa fa-spinner fa-spin"
      />
    </div>
  );
}

export default Loader;
