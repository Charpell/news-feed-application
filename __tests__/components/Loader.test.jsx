import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../../src/components/Loader';

describe('Loader Spinner', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
