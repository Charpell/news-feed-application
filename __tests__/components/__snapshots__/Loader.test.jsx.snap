// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Spinner renders a snapshot 1`] = `
<div
  className="load"
>
  Loading 
  <i
    className="fa fa-spinner fa-spin"
    style={
      Object {
        "fontSize": 24,
      }
    }
  />
</div>
`;
